arangod/Cluster/ClusterFeature.cpp

```diff
@@ -508,11 +506,27 @@ void ClusterFeature::prepare() {
     FATAL_ERROR_EXIT();
   }
 
+
+}
+
+void ClusterFeature::start() {
+
+  auto role = ServerState::instance()->getRole();
```

Comment:
Let's move this below the check for `clusterEnabled`

Reply:
Do you oversee all consequences?